### ISSUE TITLE
`znap pull` delete symlink dirs in `~/.local/bin`

### ISSUE THREAD

**Issue by soifou** [marlonrichert/zsh-snap#189]
## Describe the bug

Since [last commit](https://github.com/marlonrichert/zsh-snap/commit/c0fa1da9eee1822ab1c9428ba33fbb6c84d371dd), everytime I `znap pull` my (unrelated to znap) symlink dirs stored in `~/.local/bin` are deleted.

I'm no expert at zsh syntax but it seems the culprit is this introduced line `$bindir/*(@-^*)`.
I removed the line and symlink dirs are untouched, but I don't know the consequences in znap logic.

## Steps To Reproduce

```zsh
mkdir ~/random-dir
ln -s ~/random-dir ~/.local/bin/random-dir-symlinked

ls ~/.local/bin/ | wc -l
1

znap pull

ls ~/.local/bin/ | wc -l
0
```

## Expected behavior

My symlink dirs are untouched.
## Observed behavior

My symlink dirs are deleted.

</details>

## Environment:
 - OS: Debian
 - Terminal: Kitty
 - Zsh version: 5.9
 - Repo version: ^HEAD

**Comment by marlonrichert** [comment]
This will be solved by #179.

**Comment by soifou** [comment]
@marlonrichert 

I'm aware of that issue and I'm on latest commit. My plugins are indeed symlinked in `$XDG_STATE_HOME` but still, znap pull deletes my symlink dirs in ~/.local/bin.

Looking at `znap.pull`, `bindir` is set to `~/.local/bin` and do the following cleanup

```zsh
  local bindir=~/.local/bin
  local fdir=${XDG_DATA_HOME:-~/.local/share}/zsh/site-functions

  private -a bugs=(
    $bindir/*.*(@-*)
    $bindir/*(@-^*) # this has been added
    $bindir/*(-@)
    $fdir/*.zwc.zwc
    $fdir/*(-@)
  )
  (( $#bugs )) &&
      zf_rm $bugs[@]
```

What am I missing?

**Comment by marlonrichert** [comment]
Ah, sorry, I somehow thought I hadn't yet closed #179. 😅 

You're right, that is indeed a bug. I will fix it.

**Comment by soifou** [comment]
No problem :) and thanks for taking the time to look at it!
